src/histolab/util.py

```diff
@@ -155,7 +155,11 @@ def regions_from_binary_mask(binary_mask: np.ndarray) -> List[Region]:
     thumb_labeled_regions = label(binary_mask)
     regions = [
         Region(
-            index=i, area=rp.area, bbox=rp.bbox, center=rp.centroid, coords=rp.coords
+            index=i,
+            area=rp.area,
+            bbox=rp.bbox[:2][::-1] + rp.bbox[2:][::-1],  # from np coords to PIL coords
```

Comment:
can you explain this? It's too weird this statement 🤷🏽‍♂️

Reply:
basically, if we assume the original `rp.bbox` to be (1,2,3,4), the result of `rp.bbox[:2][::-1] + rp.bbox[2:][::-1]` would be  (2,1,4,3), so I'm flipping the order of the first two elements and then the order of the second two elements. All of this to convert from np coordinates to PIL coordinates